pkg/services/sqlstore/user.go

```diff
@@ -30,6 +45,23 @@ func (ss *SQLStore) getOrgIDForNewUser(sess *DBSession, args models.CreateUserCo
 	return ss.getOrCreateOrg(sess, orgName)
 }
 
+func (ss *SQLStore) userCaseInsensitiveLoginConflict(ctx context.Context, sess *DBSession, login, email string) error {
+	users := make([]*models.User, 0)
+	where := "LOWER(email)=LOWER(?) OR LOWER(login)=LOWER(?)"
+	login = strings.ToLower(login)
+	email = strings.ToLower(email)
```

Comment:
nit (here and elsewhere): I think you can either remove the lower-casing from `LOWER(?)` or remove `strings.ToLower` for the args, as they both seem to do the same thing.

Reply:
Removing the strings.ToLower is safer due to the potential for different implementations of LOWER()